src/pages/utilities/config-editor.tsx

```diff
@@ -0,0 +1,315 @@
+                    </select>
+                );
+            case 'show-cooldown':
+                return (
+                    <select key={name} className='form-input' id={name} defaultValue={value} onChange={handleChange}>
+                        <option value='title'>title</option>
+                        <option value='actionbar'>actionbar</option>
+                        <option value='false'>false</option>
+                    </select>
+                );
+            case 'emote-offhand-workaround':
+                return (
+                    <select key={name} className='form-input' id={name} defaultValue={value} onChange={handleChange}>
+                        <option value='disabled'>disabled</option>
+                        <option value='no-emotes'>no-emotes</option>
+                        <option value='emotes-and-offhand'>emotes-and-offhand</option>
+                    </select>
+                );
+            case 'config-version':
+                return <input key={name} className='form-input' type='text' disabled defaultValue={value.replace(/'/g, '')} />;
+            case 'metrics.uuid':
+                return <input key={name} className='form-input' id={name} type='text' disabled defaultValue={value === 'generateduuid' ? createUUID() : value} />;
+
+            default:
+                // Handle all the other inputs
+                if (value.toLowerCase() === 'true' || value.toLowerCase() === 'false') {
+                    // Boolean
+                    return (
+                        <div key={name}>
+                            <input className='checkbox' type='checkbox' id={name} defaultChecked={value.toLowerCase() === 'true'} onChange={handleChange} />
+                            <label htmlFor={name} className='switch'></label>
+                        </div>
+                    );
+                } else if (!isNaN(value)) {
+                    // Number
+                    return <input key={name} className='form-input' type='number' defaultValue={value} id={name} onChange={handleChange} />;
+                } else {
+                    // String
+                    return <input key={name} className='form-input' type='text' defaultValue={value.replace(/'/g, '')} id={name} onChange={handleChange} />;
+                }
+        }
+    }
+
+    // From https://gist.github.com/danallison/3ec9d5314788b337b682
+    function downloadString(text, fileType, fileName) {
+        const blob = new Blob([text], { type: fileType })
+
+        const a = document.createElement('a')
+        a.download = fileName
+        a.href = URL.createObjectURL(blob)
+        a.dataset.downloadurl = [fileType, a.download, a.href].join(':')
+        a.style.display = 'none'
+        document.body.appendChild(a)
+        a.click()
+        document.body.removeChild(a)
+
+        setTimeout(() => { URL.revokeObjectURL(a.href) }, 1500)
+    }
+
+    function createUUID() {
```

Comment:
is there a reason we can't use `crypto.randomUUID()` ?

Reply:
There is not, this was copied over from the previous website code. I’ll change it later